src/commands/System/Admin/reboot.ts

```diff
@@ -6,8 +6,10 @@ import { applyDescriptionLocalizedBuilder, fetchT } from '@sapphire/plugin-i18ne
 import { envParseBoolean, envParseString } from '@skyra/env-utilities';
 
 @ApplyOptions<BirthdayyCommand.Options>({
+	name: 'reboot',
 	description: 'commands/system:rebootDescription',
-	permissionLevel: PermissionLevels.BotOwner
+	permissionLevel: PermissionLevels.BotOwner,
+	preconditions: ['BotOwner']
```

Comment:
```suggestion

```

Reply:
Why removing the precondition? There is a reason why they exist right?